grid2op/tests/test_ma.py

```diff
@@ -906,6 +917,76 @@ def test_local_action_to_global_set_storage(self):
     # change_interco_status
     # set_...
 
+    def check_legal_actions(self):
+        # We create a legal action and check
+        # if _build_global_action gives the
+        # correct global_action
+        action = dict()
+        local_id = 0
+        for agent in self.ma_env.agents:
+            action[agent] = self.ma_env.action_spaces[agent]({})
+            action[agent].change_line_status = [local_id]
+        
+        self.ma_env._build_global_action(action, self.ma_env.agents)
+        
+        ref_action = self.ma_env._cent_env.action_space({})
+        ref_action.change_line_status = [
+            self.ma_env._subgrids_cls['action'][agent].line_orig_ids[local_id]
+            for agent in self.ma_env.agents
+        ]
+        
+        # We check if the global action is the wanted action
+        assert ref_action == self.ma_env.global_action
+    
+    def check_illegal_actions(self):
+        # We create an illegal action and check
+        # if _build_global_action gives the
+        # do nothing
+        action = dict()
+        local_id = 0
+        for agent in self.ma_env.agents:
+            action[agent] = self.ma_env.action_spaces[agent]({})
+            action[agent].change_line_status = [local_id, local_id+1]
+        
+        self.ma_env._build_global_action(action, self.ma_env.agents)
+        
+        ref_action = self.ma_env._cent_env.action_space({})
```

Comment:
would be easier to read if it was called "do nothing"...

Reply:
done